src/usbgroup.cpp

```diff
@@ -69,3 +69,13 @@ bool USBGroup::matchForFilter(const USBProxyFilter *filter) const
         return true;
     }
 }
+
+int USBGroup::packetCount() const
+{
+    return m_count;
+}
+
+USBPacket* USBGroup::packet(int i) const
+{
+    return &m_first[i];
```

Comment:
this looks suspicious to me

Reply:
I was thinking that USBPackets were in adjascent memory locations but it turned out they weren't... Code removed in https://github.com/lambdaconcept/usb2sniffer-qt/pull/27/commits/89523ec40e9990e5afb8ca6b0c154420675f9c7f.